gendoc.sh

```diff
@@ -68,14 +67,14 @@ if [[ "${SKIP_GIT}" == no ]]; then
     rm -rf ${ZITI_DOC_GIT_LOC}/ziti-*
   fi
   git config --global --add safe.directory $(pwd)
-  clone_or_pull "https://github.com/openziti/ziti" "ziti-cmd"
-  clone_or_pull "https://github.com/openziti/ziti-sdk-csharp" "ziti-sdk-csharp"
-  clone_or_pull "https://github.com/openziti/ziti-sdk-c" "ziti-sdk-c"
-  clone_or_pull "https://github.com/openziti/ziti-android-app" "ziti-android-app"
-  clone_or_pull "https://github.com/openziti/ziti-sdk-swift" "ziti-sdk-swift"
-  clone_or_pull "https://github.com/openziti/ziti-tunnel-sdk-c" "ziti-tunnel-sdk-c"
-  clone_or_pull "https://github.com/openziti/helm-charts" "helm-charts"
-  clone_or_pull "https://github.com/openziti-test-kitchen/kubeztl" "kubeztl"
+  clone_or_pull "https://github.com/openziti/ziti" "ziti-cmd" release-next >/dev/null
```

Comment:
why?

Reply:
Override the branch `main`. The new README files for the container images only exist in release-next.

Reply:
reverted branch override now that the files are available in `main`